### ISSUE TITLE
Bug: quiet flag appears to no longer work in 3.5.0

### ISSUE THREAD

**Issue by cablej** [mitre/hipcheck#295]
Hi,

In the latest version of hipcheck, the quiet flag no longer appears to work.

For instance, in 3.5.0:

```
# hc --version
hipcheck 3.5.0
# hc check --format json -v quiet https://github.com/mitre/hipcheck
           Analyzing GitHub repo mitre/hipcheck from https://github.com/mitre/hipcheck
                Done loading configuration and data files (0 seconds)
⏳ ⠒ resolving git repository target (pulling)     
...
```

Compared to 3.4.0:
```
# hc --version
Hipcheck 3.4.0
# hc check --format json -v quiet https://github.com/mitre/hipcheck     

{
  "repo_name": "https://github.com/mitre/hipcheck",
  "repo_head": "667d17b",
...
```

**Comment by alilleybrinker** [comment]
Thanks for the report! I think this is related to our introduction of progress reporting. We’ll get it fixed!

**Comment by patrickjcasey** [comment]
closed by #299
